### PR TITLE
Create a wrap component to wrap single tenant component

### PR DESCRIPTION
**Pull request by lburgazzoli**
**Release Note**
```release-note
NONE
```

### CODE REVIEW

**Comment by lburgazzoli** [comment]
Workaround for #495

File: camel-k-runtime-bom/pom.xml [hunk 1]
```diff
@@ -139,6 +139,11 @@
                 <artifactId>camel-kamelet</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.apache.camel.k</groupId>
+                <artifactId>camel-wrao</artifactId>
```
Comment (nicolaferraro) [inline camel-k-runtime-bom/pom.xml:144]: typo

Comment (oscerd) [inline camel-k-runtime-bom/pom.xml:144]: Wrap

**Review by oscerd — CHANGES_REQUESTED** [review]
Lgtm but the module name is wrong

**Review by nicolaferraro — CHANGES_REQUESTED** [review]
Cool, it's not so bad as workaround..

**Comment by lburgazzoli** [comment]
@oscerd @nicolaferraro mind doing a new review ?

**Review by oscerd — APPROVED** [review]

**Pull request merged**